assets/config.yml

```diff
@@ -22,61 +20,73 @@ classifierLabels:
    - "twist_drill_bit_32_165"
    - "saber_saw_blade_makita_t_300"
    - "auger_drill_bit_20_235"
+
 classifierImageSize: 384
 classifierImageChannels: 3
 classifierMean:
-   - 0.485
-   - 0.456
-   - 0.406
+  - 0.485
+  - 0.456
+  - 0.406
 classifierStd:
-    - 0.229
-    - 0.224
-    - 0.225
+  - 0.229
+  - 0.224
+  - 0.225
```

Comment:
Please keep the same indentation as before, here is one space less.

Reply:
Oops! Sorry! Fixed the config file indentation in https://github.com/ibois-epfl/TTool/pull/19/commits/d546ef36a0e35a5b28bc565d7995e39ac6f31199. Please check it out ! :)